cranelift/codegen/src/machinst/abi.rs

```diff
@@ -564,7 +564,7 @@ pub trait ABIMachineSpec {
         opcode: ir::Opcode,
         tmp: Writable<Reg>,
         callee_conv: isa::CallConv,
-        callee_conv: isa::CallConv,
```

Comment:
How is this even allowed?

Reply:
Yeah, rustc should've caught it. I'm creating an issue.